pkg/server/server.go

```diff
@@ -140,8 +142,8 @@ func (s *Server) GetEventsWG(request *tetragon.GetEventsRequest, server tetragon
 					return err
 				}
 			}
-		case <-server.Context().Done():
-			return server.Context().Err()
+		case <-s.ctx.Done():
+			return s.ctx.Err()
```

Comment:
Don't we need to check both contexts here?

I think the server.Context() can also be completed (e.g., due to timeouts in the stream)

Reply:
yep done now.